examples/universal-cnf/vppagent/cmd/config/config_endpoints.go

```diff
@@ -49,9 +51,9 @@ func NewProcessEndpoints(backend UniversalCNFBackend, endpoints []*Endpoint) *Pr
 		}
 
 		// Build the list of composites
-		compositeEndpoints := []endpoint.ChainedEndpoint{
+		compositeEndpoints := []networkservice.NetworkServiceServer{
 			endpoint.NewMonitorEndpoint(configuration),
-			NewUniversalCNFEndpoint(backend, e),
+			endpoint.NewConnectionEndpoint(configuration),
```

Comment:
Should these be swapped?

Reply:
Nope, Monitor the only part was not reversed, since it send update to monitor only after all other parties are complete.